packages/-adapter-encapsulation-test-app/app/templates/application.hbs

```diff
@@ -0,0 +1,5 @@
+{{!-- The following component displays Ember's default welcome message. --}}
+<WelcomePage />
```

Comment:
we should remove this component, we already removed the addon from package.json (same for serializer package, needs removed)